cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/DcLineSegmentConversion.java

```diff
@@ -30,32 +34,146 @@ public DcLineSegmentConversion(PropertyBag l, Context context) {
         cconverter2 = context.dc().cgmesConverterFor(iconverter2);
     }
 
+    private static double getPAc(PropertyBag p) {
+        return Double.isNaN(p.asDouble(TARGET_PPCC)) ? 0 : p.asDouble(TARGET_PPCC); // targetPpcc is the real power injection target in the AC grid in CGMES
+    }
+
+    private void updateLossFactor1(double pAC1, double poleLossP1, HvdcLine.ConvertersMode mode) {
+        if (mode.equals(HvdcLine.ConvertersMode.SIDE_1_RECTIFIER_SIDE_2_INVERTER)) {
+            iconverter1.setLossFactor((float) (poleLossP1 / pAC1) * 100);
+        } else if (pAC1 + poleLossP1 != 0) {
+            iconverter1.setLossFactor((float) (poleLossP1 / (pAC1 + poleLossP1)) * 100);
+        }
+    }
+
+    private void updateLossFactor2(double pAC2, double poleLossP2, HvdcLine.ConvertersMode mode) {
+        if (mode.equals(HvdcLine.ConvertersMode.SIDE_1_INVERTER_SIDE_2_RECTIFIER)) {
+            iconverter2.setLossFactor((float) (poleLossP2 / Math.abs(pAC2)) * 100);
```

Comment:
If side 2 is rectifier, pAC2 is positive, I think that Math.abs() is not needed.

Reply:
Same remark as above, pAC1 or pAC2 are not positive/negative depending of the stations are rectifier/inverter but if they are side 1 or 2.